site.conf

```diff
@@ -163,5 +163,10 @@
     suffix = 'nodename',      -- generate the SSID with either 'nodename', 'mac' or to use only the prefix: 'none'
     tq_limit_enabled = false, -- incompatible with Batman V. The offline SSID will only be set if there is no gateway reacheable.
   },
+
+  update_channel = { -- next branch only !!!
```

Comment:
dont know where the mistake is, it doesnt build

Reply:
#275 is the fix. Can you resolve this conversation?